### ISSUE TITLE
No longer create PR on my repo

### ISSUE THREAD

**Issue by Fred-Vatin** [google-github-actions/release-please-action#686]
### TL;DR

I used the workflow successfully many times before on the same repo. One minute it worked. Then after the last merge PR, the push are detected but the release-please won’t create PR anymore.

### Action YAML

```yaml
name: release-please

on:
  push:
    branches:
      - main
      - master
  workflow_dispatch:   

permissions:
  contents: write
  pull-requests: write

jobs:
  log:
    uses: ./.github/workflows/log.yml

  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          # need a personal access token : https://github.com/settings/tokens
          token: ${{ secrets.RELEASE_PLEASE }}
          # node allows to access package.json to read and bump version
          release-type: node
          changelog-types: |
            [
              {"type":"feat","section":"🚀 Features","hidden":false},
              {"type":"fix","section":"🐛 Bug Fixes","hidden":false},
              {"type":"perf","section":"🧰 Miscellaneous","hidden":false},
              {"type":"refactor","section":"🧰 Miscellaneous","hidden":false},
              {"type":"chore","section":"🧰 Miscellaneous","hidden":false},
              {"type":"build","section":"🧰 Miscellaneous","hidden":false},
              {"type":"deps","section":"🧰 Miscellaneous","hidden":false}
            ]
          pull-request-header: ":robot: I have created a release"

      - run: echo "🍏 This job's status is ${{ job.status }}."
```

### Log output

Please go to https://github.com/Fred-Vatin/never-be-lost/actions/workflows/release-please.yml

You can compare the working runs before [#20](https://github.com/Fred-Vatin/never-be-lost/actions/runs/3797526940) and the failed ones after [#20](https://github.com/Fred-Vatin/never-be-lost/actions/runs/3797588256).

The working runs always created a PR. From [#21](https://github.com/Fred-Vatin/never-be-lost/actions/runs/3797588256), it refuses.

### Additional information

I tested release-please versions from 3.5.1 to 3.7.1.

I didn’t touch the workflow.

I may have refused a PR before but it didn’t stop release-please to produce a new one.

I may have deleted the release-please branch after a successful merge PR. I don’t know it could be related.

**Comment by Fred-Vatin** [comment]
Ok I got it. It doesn’t create PR if there is no conventional commits triggering at least fix changes.
